storage/pebble/registers.go

```diff
@@ -61,7 +69,14 @@ func (s *Registers) Store(
 	height uint64,
```

Comment:
I want to discuss this, I don't think updating the DB `latestHeight` in `Store` when we index the entire checkpoint file, it will have to load it in batches, the first call to `Store` should not update `latestHeight` or `firstHeight` at the pebble level